modules/ROOT/pages/_partials/generic-local-mobile-dev-envir-reqs.adoc

```diff
@@ -54,5 +54,5 @@ You need the following installed on your system:
 * [x] (Optional) The latest version of http://openjdk.java.net/install/index.html[OpenJDK, window="_blank"], if you would like to install the showcase app on an Android device or emulator
 * [x] (Optional) The latest version of XCode and command line tools for XCode available from https://developer.apple.com/download/more/[Apple, window="_blank"], if you would like to install the showcase app on an iOS device or emulator
 
-For information on how to set up your local development environment for Xamarin, see link:https://developer.xamarin.com/getting-started/[Xamarin, window="_blank"].
+For information on how to set up a local Xamarin development environment, see link:https://developer.xamarin.com/getting-started/[Xamarin, window="_blank"].
```

Comment:
see the link:https://developer.xamarin.com/getting-started/[Xamarin documentation, window="_blank"].